core-web/libs/edit-content/src/lib/fields/dot-edit-content-category-field/components/dot-category-field-search-list/dot-category-field-search-list.component.ts

```diff
@@ -162,4 +193,41 @@ export class DotCategoryFieldSearchListComponent implements AfterViewInit, OnDes
             this.$scrollHeight.set(`${containerHeight}px`);
         }
     }
+
+    /**
+     * Retrieves the message configuration based on the current component status.
+     *
+     * @private
+     * @returns {PrincipalConfiguration | null} Returns the message configuration, or null if no configuration is found.
+     */
+    private getMessageConfig(): PrincipalConfiguration | null {
+        const configKey =
+            this.$status() === ComponentStatus.ERROR ? ComponentStatus.ERROR : 'empty';
+        const { title, icon, subtitle } = this.messageConfigs[configKey];
+
+        return {
+            title: this.#messageService.get(title),
+            icon: this.#messageService.get(icon),
+            subtitle: this.#messageService.get(subtitle)
+        };
+    }
+
+    /**
+     * An object containing the configuration for different types of messages.
+     */
+    private readonly messageConfigs: Record<
```

Comment:
@oidacra I think this object could be a const outside the component or in case to keep it inside the component it should use `#` because it is private.

Reply:
moved to a constant.